src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/book_store?serverTimezone=UTC";
```

Comment:
The `DB_URL` is incomplete, which will lead to a `SQLException` when trying to establish a database connection. Make sure to provide the full JDBC URL including the database name and any necessary parameters.